QUEST.tsv

```diff
@@ -1895,7 +1895,7 @@ QUEST_20150428_001898	The Letter that is Ripping into Pieces
 QUEST_20150428_001899	Ripped Diary
 QUEST_20150428_001900	The Last Record
 QUEST_20150428_001901	$The Abandoned Diary
-QUEST_20150428_001902	$The Goddess left many things to the first Paladin: {nl}the threat of the demon king, Gesti, and the mission to protect. {nl}She also mentioned that the savior would come.
+QUEST_20150428_001902	$The Goddess left many things to the first Paladin: {nl}The threat of the Demon Queen, Gesti, and the mission to protect. {nl}She also mentioned that the savior would come.
```

Comment:
Should there be a pause/comma after Demon Queen? Not only is it a bit confusing, look and sound odd, but it also implies that there are three things. The Demon Queen, Gesti, and the mission to protect. 

When you read the context around the sentence, it also further implies there are more then two things, which confuses the sentence structure even more.